sentinel-core/src/main/java/com/alibaba/csp/sentinel/log/RecordLog.java

```diff
@@ -15,80 +15,74 @@
  */
 package com.alibaba.csp.sentinel.log;
 
-import java.util.Iterator;
-import java.util.ServiceLoader;
+import com.alibaba.csp.sentinel.log.jul.JavaLoggingAdapter;
 
-/***
- * The basic logger for vital events.
+/**
+ * The basic biz logger of Sentinel.
  *
  * @author youji.zj
+ * @author Eric Zhao
  */
-public class RecordLog extends LogBase {
-    private static com.alibaba.csp.sentinel.log.Logger log = null;
+public class RecordLog {
+
+    public static final String RECORD_LOGGER_NAME = "sentinelRecordLogger";
+    public static final String DEFAULT_RECORD_LOG_FILENAME = "sentinel-record.log";
```

Comment:
The name of final variables are not consistent with `CommandCenterLog` class, as they are in context of `RecordLog`, maybe the same as `RecordLog`'s `LOGGER_NAME`、`DEFAULT_LOG_FILENAME` is better? Seems `DEFAULT_LOG_FILENAME` can be `LOG_FILENAME`.

Reply:
Good idea. I've polished the code, please check.

Reply:
OK, got it. Now `LogBase` is not a parent class, maybe another class name is better, like `LogConfig` or sth else. Besides, I found that the config parameter which can be customized by user is scattered in different classes(by global seaching "csp.sentinel." in sentinel-core module), if it is better put them into one config class, to make config clearly and convenient to found. And this need careful consideration and discussion, and I think maybe we can do it in the future.